server/planning/feeding_services/onclusive_api_service.py

```diff
@@ -101,18 +101,13 @@ def _update(self, provider, update):
                 logger.info("Ingesting content: {} ...".format(str(between_event_response.content)[:4000]))
 
                 if hasattr(parser, "parse_http"):
```

Comment:
are there actually both options possible? imo we should limit onclusive to single parser as we do for rss